observability/test/manual_deploy.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -ex
+cd "$(dirname "$0")"
+
+export PROJECT=`gcloud config get-value project`
+export PROJNUM=`gcloud projects describe $PROJECT --format="value(projectNumber)"`
+export SERVER_IMAGE=gcr.io/microsvcs-testing/grpc-observability/testing/integration-dev-go:1.53.0-dev
```

Comment:
I see multipole references to 1.53 in this PR. We should probably make these references to a single place where the gRPC version is defined.

Reply:
Yes I added a TODO to hopefully remove the need for this constant.